plutus-contract/src/Wallet/API.hs

```diff
@@ -91,7 +91,7 @@ payToPaymentPublicKeyHash range v pk = do
     utx <- either (throwError . PaymentMkTxError)
                   pure
                   (Constraints.mkTx @Void mempty constraints)
-    let adjustedUtx = Constraints.adjustUnbalancedTx utx
+    let adjustedUtx = Constraints.adjustUnbalancedTx minAdaTxOut utx
```

Comment:
The only place where `adjustUnbalancedTx` from `Constraints` being used. It's hard to import `adjustUnbalancedTx` from `Contract`, there are circular dependency modules.

Reply:
Makes sense 👍